api/src/crate/benchapi/application.py

```diff
@@ -0,0 +1,120 @@
+# vi: set fileencoding=utf-8
+# -*- coding: utf-8; -*-
+
+import json
+import toml
+from time import mktime, strptime
+from datetime import datetime
+from flask import Flask, g as app_globals, make_response, jsonify, request
+from flask.ext.restful import Resource
+from flask.ext.cors import CORS
+from crate.client import connect
+from crate.client.exceptions import ProgrammingError
+
+
+app = Flask(__name__)
+# apply CORS headers to all responses
+CORS(app)
+
+
+class CrateResource(Resource):
+
+    def __init__(self):
+        super().__init__()
+        self.cursor = self.connection.cursor()
+
+    def __del__(self):
+        self.cursor.close()
+
+    @property
+    def connection(self):
+        if not 'conn' in app_globals:
+            app_globals.conn = connect(app.config.get('crate_hosts', []),
+                                       error_trace=app.config.get('debug', False))
+        return app_globals.conn
+
+    def error(self, message, status=404):
+        return (dict(
+            error=message,
+            status=status,
+        ), status)
+
+    def convert(self, description, results):
+        cols = [c[0] for c in description]
+        return [dict(zip(cols, r)) for r in results]
+
+
+class Result(CrateResource):
+    """
+    Resource for doc.benchmarks
+    Supported method: GET
+    """
+
+    def get(self, group):
+        param_from = request.args.get('from')
+        param_to = request.args.get('to')
+        params = []
+        where_clause = []
+        time_from = datetime.now()
```

Comment:
probably doesn't matter because the time is stripped anway - but I think the build hash time in Crate is utc - so might make sense to use `utcnow` here.